core/controllers/dashboard.py

```diff
@@ -204,6 +204,7 @@ def _round_average_ratings(rating):
                     'created_on': utils.get_time_in_millisecs(
                         collection_summary.collection_model_created_on),
                     'status': collection_summary.status,
+                    'node_count': collection_summary.node_count,
```

Comment:
Note the TODO at the top, in lines 194-195. Do you think it's possible to reuse the given method?